src/main/java/cz/cvut/kbss/termit/persistence/dao/VocabularyDao.java

```diff
@@ -49,7 +49,8 @@
     private final ApplicationContext context;
 
     @Autowired
-    public VocabularyDao(EntityManager em, Configuration config, DescriptorFactory descriptorFactory, ChangeRecordDao changeRecordDao, ApplicationContext context) {
+    public VocabularyDao(EntityManager em, Configuration config, DescriptorFactory descriptorFactory,
```

Comment:
why this reformatting?

Reply:
Hopefully fixed with updated .editorconfig.